forms-flow-web/src/apiManager/services/dashboardsService.js

```diff
@@ -75,70 +50,44 @@ export const fetchGroups = () => {
   };
 };
 
-// dashboards property should be cleaned before updating the store
-export const cleanGroups = (groups) => {
-  if (groups?.length === 0) {
-    return groups;
-  }
-
-  let newGroups = [...groups];
-
-  for (let group of newGroups) {
-    group.dashboards = getCleanedDashboards(group.dashboards);
-  }
-
-  return newGroups;
+export const updateAuthorization = (data) => {
+  return (dispatch) => {
+    httpPOSTRequest(API.DASHBOARD_AUTHORIZATION, data)
```

Comment:
Not specific to this change; but can't we handle the timeout globally? Like wrap the http request and add time out there ?

Reply:
The timeout looks hacky to me, will replace it with toast